docs/source/install.rst

```diff
@@ -266,10 +266,11 @@ Create a build directory and step into it::
     mkdir build
     cd build
 
-From the build directory you can now configure CMake and build the binaries::
+From the build directory you can now configure CMake, build and install the binaries::
 
     cmake ..
     cmake --build .
+    cmake --install .
```

Comment:
the "install" step still needs to look like this:
```suggestion
    cmake --build . --target install
```
(however, your suggestion will be used when CMake 3.15 is the minimum, since that's the version when that switch was introduced). Nearly there!

Reply:
So that's how it's done. Thank you very much @mwtoews for your help!